### ISSUE TITLE
Karma doesn't reload and doesn't show any feedback about the test status.

### ISSUE THREAD

**Issue by phedinkus** [Swiip/generator-gulp-angular#525]
I have a clean install of the the generator on node v0.12.2. I ran the tests using `gulp test:auto`. I see the following on the karma page:
![screen shot 2015-05-01 at 12 28 32 pm](https://cloud.githubusercontent.com/assets/405235/7433262/9f300098-effd-11e4-91e9-495e50e22823.png)
and from the debug console: 
![screen shot 2015-05-01 at 12 29 04 pm](https://cloud.githubusercontent.com/assets/405235/7433272/b46f91bc-effd-11e4-9206-eb0f33836ce4.png)

Maybe I'm wrong, but I expect if I change the test for the main controller like so: 
```
expect(scope.awesomeThings.length < 5).toBeTruthy();
```
that it should fail. The build process runs in my console but none of the output in Karma changes in my browser. Even if I refresh. 

If I restart `gulp test:auto` with the failing test the output in the debug console is correct: 
![screen shot 2015-05-01 at 12 32 22 pm](https://cloud.githubusercontent.com/assets/405235/7433326/2a37cc48-effe-11e4-8d0e-3226035d732e.png)
The other main Karma screenshot is the same as the first (which also seems wrong).

**Comment by phedinkus** [comment]
I also noticed I wasn't getting any output from Karma in console. 
![screen shot 2015-05-01 at 12 56 28 pm](https://cloud.githubusercontent.com/assets/405235/7433648/8a1e924c-f001-11e4-8964-7ff297a07634.png)

I took a look at the karma config and noticed that `autoWatch` is set the false. I changed it and reran `gulp test:auto` and now I have output in my console: 
![screen shot 2015-05-01 at 12 57 09 pm](https://cloud.githubusercontent.com/assets/405235/7433660/aa521796-f001-11e4-8fe7-0bb4a58506db.png)
It also now reruns when I change my tests: 
![screen shot 2015-05-01 at 12 58 07 pm](https://cloud.githubusercontent.com/assets/405235/7433674/c1f51d30-f001-11e4-99cd-e565cffc324c.png)

**Comment by Swiip** [comment]
Please check with the master version because I don't reproduce you problem. I change the `>` by a `<` and it rerun.

```
INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS
WARN [watcher]: Pattern "/Users/Swiip/Workspace/Sandbox/gulp-angular/src/**/*.mock.js" does not match any file.
INFO [PhantomJS 1.9.8 (Mac OS X)]: Connected on socket 5vqs-lU98U10a2utc1Gl with id 16829350
PhantomJS 1.9.8 (Mac OS X): Executed 1 of 1 SUCCESS (0.002 secs / 0.014 secs)
[17:02:48] Starting 'scripts'...
INFO [watcher]: Changed file "/Users/Swiip/Workspace/Sandbox/gulp-angular/src/app/main/main.controller.spec.js".

/Users/Swiip/Workspace/Sandbox/gulp-angular/src/app/main/main.controller.spec.js
  line 1  col 1  Use the function form of "use strict".

  ⚠  1 warning

[17:02:48] all files 3.33 kB
[17:02:48] Finished 'scripts' after 94 ms
PhantomJS 1.9.8 (Mac OS X) controllers should define more than 5 awesome things FAILED
	Expected false to be truthy.
	    at /Users/Swiip/Workspace/Sandbox/gulp-angular/src/app/main/main.controller.spec.js:16
	    at invoke (/Users/Swiip/Workspace/Sandbox/gulp-angular/bower_components/angular/angular.js:4203)
	    at workFn (/Users/Swiip/Workspace/Sandbox/gulp-angular/bower_components/angular-mocks/angular-mocks.js:2436)
PhantomJS 1.9.8 (Mac OS X): Executed 1 of 1 (1 FAILED) ERROR (0.002 secs / 0.019 secs)
```

**Comment by phedinkus** [comment]
I reinstalled the generator, created a new project and ran `gulp test:auto` and got a completely new error:
```
▶ gulp test:auto
[11:24:32] Using gulpfile ~/Code/client/gulpfile.js
[11:24:32] Starting 'markups'...
[11:24:32] Starting 'scripts'...
[11:24:32] Starting 'styles'...
[11:24:33] Finished 'markups' after 796 ms
[11:24:33] gulp-inject Nothing to inject into index.less.
[11:24:33] all files 3.4 kB
[11:24:33] Finished 'scripts' after 918 ms
[11:24:35] Finished 'styles' after 2.3 s
[11:24:35] Starting 'inject'...
[11:24:35] gulp-inject 1 files into index.html.
[11:24:35] gulp-inject 3 files into index.html.
[11:24:35] Finished 'inject' after 105 ms
[11:24:35] Starting 'watch'...
[11:24:35] Finished 'watch' after 19 ms
[11:24:35] Starting 'test:auto'...
/Users/kt/Code/client/gulp/unit-tests.js:39
          .concat(_.pluck(files, 'path'))
          ^
TypeError: Cannot read property 'concat' of undefined
    at /Users/kt/Code/client/gulp/unit-tests.js:39:11
    at ConcatStream.<anonymous> (/Users/kt/Code/client/node_modules/concat-stream/index.js:36:43)
    at ConcatStream.emit (events.js:129:20)
    at finishMaybe (/Users/kt/Code/client/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:460:14)
    at endWritable (/Users/kt/Code/client/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:469:3)
    at ConcatStream.Writable.end (/Users/kt/Code/client/node_modules/concat-stream/node_modules/readable-stream/lib/_stream_writable.js:436:5)
    at DestroyableTransform.onend (/Users/kt/Code/client/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:523:10)
    at DestroyableTransform.g (events.js:199:16)
    at DestroyableTransform.emit (events.js:129:20)
    at /Users/kt/Code/client/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:965:16
```

**Comment by phedinkus** [comment]
To clarify, I didn't touch any of the generated project.

**Comment by Swiip** [comment]
Sorry for the delay but fixed for long but still not released:  #480

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada:
Your issue should be fixed. I close but if you can retry your case with a new fresh install of generator you are welcome. Any feedback would be appreciated (positive comments too ^_^).
